### ISSUE TITLE
Counter on top of pages count the number of bugs instead of the number of patches

### ISSUE THREAD

**Issue by mounirlamouri** [fitzgen/bugzilla-todos#9]
In other words, if I have to review 7 patches contained in two bugs, my review tab will show "2" instead of "7". The same for all tabs I believe.

**Comment by harthur** [comment]
Good catch, it seems like it should show number of patches for the review tab, and affects Nags and Respond tab as well.
